### ISSUE TITLE
Roshade not working

### ISSUE THREAD

**Issue by Sourtcat0** [bituq/Roshade#40]
I just can't get it to work I Followed lots of steps on YouTube and they all don't work I don't know how to use them

**Comment by 0hStormy** [comment]
This is because Roblox is now a 64-bit application, the developers will have to update roshade to work with 64-bit

**Comment by thijsi123** [comment]
Damn, I sure hope that someone will add support then!

**Comment by Sourtcat0** [comment]
Yes I know lots of people say its because its 64 bit but I want make sure its not something else.
